src/Creator.php

```diff
@@ -21,11 +26,13 @@ class Creator implements CreatorInterface
     /**
      * Creator constructor.
      *
-     * @param bool  $limitUpscaling
-     * @param Color $background
+     * @param bool       $limitUpscaling
+     * @param SvgImagine $svgImagine
+     * @param Color      $background
      */
-    public function __construct($limitUpscaling = true, Color $background = null)
+    public function __construct($limitUpscaling = true, SvgImagine $svgImagine = null, Color $background = null)
```

Comment:
Hmm I didn't want to go past background since it wasn't provided in service provider.